test/Interop/Cxx/class/nonescapable-lifetimebound.swift

```diff
@@ -120,6 +121,21 @@ CaptureView getCaptureView(const Owner& owner [[clang::lifetimebound]]) {
 // CHECK: sil [clang CaptureView.captureView] {{.*}} : $@convention(cxx_method) (View, @lifetime(copy 0) @inout CaptureView) -> ()
 // CHECK: sil [clang CaptureView.handOut] {{.*}} : $@convention(cxx_method) (@lifetime(copy 1) @inout View, @in_guaranteed CaptureView) -> ()
 
+// CHECK-ADDRESSABLE: sil [clang makeOwner] {{.*}}: $@convention(c) () -> Owner
```

Comment:
@jckarter do you know what would be the best way to test if `@addressable` is properly applied? I do not see this reflected in the emitted SIL. Is this an oversight or intentional?

Reply:
We talked offline, I added a test.